theia-electron/package.json

```diff
@@ -0,0 +1,64 @@
+{
+    "private": true,
+    "name": "theia-electron",
+    "version": "0.3.16",
```

Comment:
When I build for Linux, I get a .deb package: `theia-electron_0.3.16_amd64.deb`. I assume that the `0.3.16` part of the Debian package name comes from here? The same version is used in the metadata of the package:

![image](https://user-images.githubusercontent.com/25749063/49178853-def58180-f31e-11e8-8ab2-be21680d1320.png)

It would be desirable for the exact `next` version to be used instead, since that version is used to determine if a given .deb package is newer than the one currently installed.